Improves the service descriptor file

rodrigorigotti opened the pull request:
Improves the service descriptor file so a faulty call is corrected and several common structutes are moved to a separate model.

lcobucci commented:
@rodrigorigotti I honestly have no context here so I'm unable to give any review

rodrigorigotti commented:
@lcobucci this is the PHP SDK for our Public API. There's a descriptor file which states what are the endpoints of the API and the parameters for each one of them.

There was a bug in the `WebsiteCampaignResultItems` model so the customers couldn't use the `GetWebsiteCampaignResults` command. I made the fix and also removed some of the redundancies in the description file so it's smaller and easier to understand.

rdohms reviewed (changes requested):
Please split the commits, its not clear what the bug was and what the reformatting is.

rodrigorigotti committed:
Fix an attribute in the service description

so the `GetWebsiteCampaignResults` command will no longer return an
empty array even if there are results to be fetched

rodrigorigotti committed:
Simplifies the description file's structure

so it's easier to read, since all commands share the same response
structure and possible error messages

rdohms commented on src/Usabilla/API/Description/UsabillaDescription.php:
```diff
@@ -226,139 +175,7 @@ public function __construct()
                         'lastTimestamp' => [ '$ref' => 'LastTimestampAttribute' ],
                     ],
```
does this not lose the item definition?

rodrigorigotti replied:
This model already existed, it wasn't just being used here.

rdohms replied:
yes but the definitions you deleted had a `'items'` key you that is now gone?

rdohms replied:
nevermind, github was confusing me.
